### PR TITLE
Document `weave status ipam`

### LINKED ISSUE #1876: Document `weave status ipam`
In `weave` usage and ipam and/or troubleshooting web page.
Implementation was in #1702.

### PR DESCRIPTION
**Pull request by awh**
Doc preview available at

https://dev-weavewww.pantheonsite.io/docs/net/1876-weave-status-ipam-docs/ipam/troubleshooting-ipam/ once CI build finishes.

Fixes #1876.

### CODE REVIEW

**Comment by awh** [comment]
CC @abuehrle

File: site/ipam/troubleshooting-ipam.md [hunk 1]
```diff
@@ -43,3 +43,12 @@ the total allocation range and default subnet. Columns are as follows:
       for more
 * 'Range' - total allocation range set by `--ipalloc-range`
 * 'DefaultSubnet' - default subnet set by `--ipalloc-default-subnet`
+
+Per-peer allocation statistics and reachability can be obtained with
```
Comment (rade) [inline site/ipam/troubleshooting-ipam.md]: Calling this "*allocation* statistics" is misleading. We, quite consistently, refer to "allocations" when talking about assigning IPs to containers, not when dividing the IP range between peers, which, iirc, is what the figures here refer to. In ipam.md#persistence we refer to this information as "The division of the IP allocation range amongst peers".

Also the "unreachable!" deserves elaboration and reference to the docs for `rmpeer`.

Generally, we need to think about what a "good" and "bad" output would look like here, and guide the user accordingly.

File: site/ipam/troubleshooting-ipam.md [hunk 2]
```diff
@@ -43,3 +43,24 @@ the total allocation range and default subnet. Columns are as follows:
...
+```
+$ weave status ipam
+00:00:00:00:00:01(one)      349526 IPs (33.3% of total)
+00:00:00:00:00:02(two)      349525 IPs (33.3% of total)
+00:00:00:00:00:03(three)    349525 IPs (33.3% of total) - unreachable!
+```
+
+Columns are as follows:
+
+* Peer Name and Nickname
```
Comment (bboreham) [inline site/ipam/troubleshooting-ipam.md]: Do we define "Peer Name" anywhere?

Comment (bboreham) [inline site/ipam/troubleshooting-ipam.md]: #2266 covers it, but we should try to have a specific definition that we can link to where it's used like this.

**Pull request merged**